website/api/trusted-setup-ceremony.md

```diff
@@ -17,18 +17,18 @@ There are multiple steps to verifying the security of a trusted setup ceremony.
 
 The circuit we are securing is the RISC Zero STARK Verify circuit, which is open source and [available on GitHub][stark-verify-circom] (along with [a short library][risc0-circom-library] it depends on).
 
-To ensure that the circuit itself does not have security holes, we have used a mixture of good software engineering practices, internal security reviews, and external audits. We have published these audits: you can read more details and the audits themselves [here][audits-readme].
+To ensure that the circuit itself does not have security holes, we have used a mixture of good software engineering practices, internal security reviews, and external audits. We have published these audits: [you can read more details and the audits themselves][audits-readme].
 
 ## The Transcript Matches the Circuit
 
-Our ceremony transcript is included in the `zkey` published on ceremony.pse.dev in the "Download ZKey" tab of the [RISC Zero STARK-to-SNARK Prover page][pse-risc0-ceremony]. (We mirror this file [here][zkey-mirror], and have a compressed version [here][zkey-compressed]) You can verify it matches the circuit using Circom and snarkjs:
+Our ceremony transcript is included in the `zkey` published on ceremony.pse.dev in the "Download ZKey" tab of the [RISC Zero STARK-to-SNARK Prover page][pse-risc0-ceremony]. ([mirror][zkey-mirror]/[compressed version][zkey-compressed]) You can verify it matches the circuit using Circom and snarkjs:
```

Comment:
This change looks sus to me...

Reply:
This one is a personnal pet peeve of mine that maybe doesn't belong in this PR 😬 

But basically using something like `You can download it [here]` is always inferior to just sauing `Download thing`

Basically don't label things with `here`, which is non-descriptive and small. Use the thing you're linking to as the labels. Hopefully I'm making sense lol

Reply:
ya i hear you on avoiding `here` as a label. 

let's save this for a future PR; there are formatting issues with your proposed revisions and this PR is already massive.

Reply:
Sounds good, reverted ✅